public/static/docs/command-reference/checkout.md

```diff
@@ -179,7 +179,7 @@ outs:
   path: model.pkl
 ```
 
-But if you check `model.pkl`, the file hash is still the same:
+But if you check `model.pkl`, the file checksum is still the same:
```

Comment:
it's actually better to use `hash`. Unfortunately, for some reason we call the checksum in the code and it affected even interface of the DVC, but this is not a correct term. Would prefer to change it into other direction.

Reply:
OK, I can revert but I don't understand why "checksum" is incorrect?

Reply:
Is it because we're not using the hashes mainly for file integrity verification?

Reply:
yes, you got it right. we use them as a key in the DVC cache (which is a key/value store) and it's important for us for these "checksums" to be unique for files that differ. That's why we don't properly support HDFS, for example - they provide checksums, not md5 or sha1 hash.

Reply:
I'm working on changing all these from "checksum" to "hash" (except in `dvc` command output samples) but warning: it's going to be another 20-file or so commit...

Note that we will then have "(Git) commit SHA hash" vs. "file hash"/"hash value" or just "hash" throughout many docs. Hopefully it's not confusing.

Reply:
@shcheklein this change will be big. Let's have it in a separate PR? It will be too overwhelming for you to review along with this PR. I have it in a separate branch now, still wrapping it up.

Reply:
OK: Extracted to #962 which includes everything in this branch, so no sense looking at the file changes until after we merge this one. Thanks

Reply:
sure! probably better to revert the related changes in this one?

Reply:
OK, reverted in a8a7c1dc.